test/mock/mock-arduino.hpp

```diff
@@ -8,6 +8,8 @@
 #include <string>
 #include <vector>
 
+#define SERIAL_RX_BUFFER_SIZE 79
```

Comment:
Is 79 special in any way? Just curious.

Reply:
it's the year I was born. :smile:

Reply:
Amazing that's the Chinese year of the taco.

Reply:
lol. :100: